lib/arelastic/builders/query_builder.rb

```diff
@@ -0,0 +1,48 @@
+module Arelastic
+  module Builders
+    module QueryBuilder
+      MACROS_TO_ARELASTIC = {
+        bool:         Arelastic::Queries::Bool,
+        ids:          Arelastic::Queries::Ids,
+        match_all:    Arelastic::Queries::MatchAll,
+        match_none:   Arelastic::Queries::MatchNone,
+        match_phrase: Arelastic::Queries::MatchPhrase,
+        prefix:       Arelastic::Queries::Prefix,
+        term:         Arelastic::Queries::Term,
+        terms:        Arelastic::Queries::Terms
+      }
+
+      def self.included(base)
+        base.class_eval do
+          extend ClassMethods
+        end
+      end
+
+      module ClassMethods
+        MACROS_TO_ARELASTIC.each do |macro, klass|
+          define_method macro do |*args|
+            klass.new(*args)
+          end
+        end
+
+        def fuzzy(field_name, value)
+          Arelastic::Queries::Match.new(
+            field_name,
+            query: value,
+            operator: 'and',
+            prefix_length: 3,
+            fuzziness: "AUTO"
+          )
+        end
```

Comment:
This is very specific to a particular use case. Also there is a es native [`fuzzy query`](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/query-dsl-fuzzy-query.html) that overlaps with this name.

Reply:
Yeah this is bad and doesn't belong in this gem.